### ISSUE TITLE
High values for navigationTiming marks

### ISSUE THREAD

**Issue by moix** [elastic/apm-agent-rum-js#817]
Hi, I noticed that rum js agent is reporting huge timing marks every now and then, see some docs reporting high values:

![Screenshot from 2020-06-19 10-10-27](https://user-images.githubusercontent.com/569132/85112261-a7f30380-b215-11ea-89df-020024f92b4d.png)

Note I'm still on 4.1.2 as shown in the screenshot, planing to upgrade but reading the release notes can't see anything anything related to it.

**Comment by vigneshshanmugam** [comment]
@moix Thanks for opening the issue, I was investigating a similar bug in Safari and created this one #818. May be both would be related, Is it possible for you to share the transaction marks? You can mock the data and share whether the values are in different timestamps.

**Comment by moix** [comment]
Hi @vigneshshanmugam  sure I can provide you details if needed, let me know if following works? I can find different user agents if you want

```
{
  "_index": "apm-7.5.1-transaction-2020.06.23",
  "_type": "_doc",
  "_id": "VZS4HIBsorMvpdyDFosP",
  "_version": 1,
  "_score": null,
  "_source": {
    "agent": {
      "name": "js-base",
      "version": "4.1.2"
    },
    "processor": {
      "name": "transaction",
      "event": "transaction"
    },
    "url": {
      "path": "/",
      "original": "url original edited",
      "scheme": "https",
      "domain": "url domain edited"
    },
    "labels": {
      "brand_code": "brand code edited",
      "language": "en",
      "channel_type": "mobile"
    },
    "tags": [
      "beats_input_raw_event",
      "urlparsed",
      "_grokparsefailure"
    ],
    "observer": {
      "hostname": "apm-server-processor-1",
      "id": "9b98e154-774c-4df2-8f26-6d244eec75b6",
      "type": "apm-server",
      "ephemeral_id": "bdf1cfa5-fce7-4667-b275-5b448d2f946d",
      "version": "7.5.1",
      "version_major": 7
    },
    "trace": {
      "id": "8512ef58f7bd9a3810cd962fcb8b2891"
    },
    "@timestamp": "2020-06-23T08:36:14.340Z",
    "ecs": {
      "version": "1.1.0"
    },
    "service": {
      "name": "Service Name Edited",
      "language": {
        "name": "javascript"
      }
    },
    "@version": "1",
    "client": {
      "geo": {
        "region_iso_code": "AZ",
        "continent_name": "NA",
        "city_name": "Phoenix",
        "country_iso_code": "US",
        "country_name": "United States",
        "region_name": "Arizona",
        "location": {
          "lon": "-111.9956",
          "lat": "33.4656"
        }
      },
      "ip": "ip edited"
    },
    "user_agent": {
      "original": "Mozilla/5.0 (iPhone; CPU iPhone OS 13_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Mobile/15E148 Safari/604.1",
      "os": {
        "name": "iOS",
        "version": "13.5",
        "full": "iOS 13.5"
      },
      "version": "13.1.1",
      "device": {
        "name": "iPhone"
      }
    },
    "transaction": {
      "duration": {
        "us": 3767000
      },
      "name": "transaction name edited",
      "marks": {
        "agent": {
          "domInteractive": 1592901370500,
          "domComplete": 1592901371459,
          "renderTime": 825
        },
        "navigationTiming": {
          "domInteractive": 1592901370500,
          "domContentLoadedEventStart": 1592901370500,
          "domComplete": 1592901371459,
          "domContentLoadedEventEnd": 1592901370500,
          "domLoading": 1592901370129
        }
      },
      "id": "f0ef3fa99004c1f1",
      "page": {
        "referer": "referer edited"
      },
      "span_count": {
        "started": 0
      },
      "type": "page-load",
      "sampled": false
    },
    "timestamp": {
      "us": 1592901374340709
    }
  },
  "fields": {
    "@timestamp": [
      "2020-06-23T08:36:14.340Z"
    ]
  },
  "highlight": {
    "agent.name": [
      "@kibana-highlighted-field@js-base@/kibana-highlighted-field@"
    ]
  },
  "sort": [
    1592901374340
  ]
}
```

**Comment by vigneshshanmugam** [comment]
Also, could you please check if the issue is from webkit based browsers?

**Comment by vigneshshanmugam** [comment]
@moix  Thanks for sharing the details. Super helpful. Wow, its the same bug I was seeing in Webkit - https://bugs.webkit.org/show_bug.cgi?id=168057.

I have a PR open that should fix this issue #819

**Comment by moix** [comment]
awesome, many thanks!
